UPGRADE-15.0.md

```diff
@@ -861,6 +861,14 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
 -   ParameterColorValueDataFixture is added to handle assigning hex values to color parameters
 -   see #project-base-diff to update your project
 
+#### update easy-coding-standard to version 12.2 ([#3192](https://github.com/shopsys/shopsys/pull/3192))
+
+-   update configuration file to new version
```

Comment:
once again, it is very sad that we have no upgrade guide to link to. I am afraid people will be quite sad but I do not feel like writing upgrade notes for another package

Reply:
Yes... I hope the project-base diff give at least a general idea about necessary changes